front/components/sparkle/navigation.tsx

```diff
@@ -112,39 +114,50 @@ export const subNavigationAdmin = ({
 
   nav.push({
     id: "conversation",
-    label: "Conversations",
+    label: null,
```

Comment:
Why null here ?

Reply:
To match the figma with no section title for these 3 ones :+1: